### ISSUE TITLE
Flip Attributes: Offset, Size

### ISSUE THREAD

**Issue by ax3l** [ComputationalRadiationPhysics/libSplash#162]
We just changed the file format in a full version to introduce `h5py` compatible `bool` values in #153.

With the upcoming release of `1.3` we could use the change and also
  - flip the attributes for offsets and sizes

to be correctly in `z,y,x` order too (we also flip the coordinates of the data sets). That would reduce the confusion for users while reading our data with other tools, e.g., with `h5py`.

Might also need:
  - [ ] an update of our tools for `xdmf`

Spotted by @psychocoderHPC  - @f-schmitt-zih do you agree?

**Comment by f-schmitt-zih** [comment]
Of course, sounds like a good change! I wouldn't call it a bug, though.

**Comment by ax3l** [comment]
True, it's more of a refactoring :)

**Comment by ax3l** [comment]
an other day where I tried to refactor this and it's just super confusing and we barely have [2D data sets](https://github.com/skuschel/postpic/blob/26feba4a33d59e456354f7f21ba81f6cdf69d9b4/postpic/datareader/picongpuh5.py#L74) in the examples/tests...

**Comment by ax3l** [comment]
moving this to `2.0`, it just hurts too look at it.

Working PR's for 1, 2 and 3D data sets are very welcome.
